web_src/js/components/DiffCommitSelector.vue

```diff
@@ -94,7 +94,8 @@ export default {
     focusElem(elem, prevElem) {
       if (elem) {
         elem.tabIndex = 0;
-        prevElem.tabIndex = -1;
+        if (prevElem)
+          prevElem.tabIndex = -1;
```

Comment:
```suggestion
        if (prevElem) prevElem.tabIndex = -1;
```

I prefer either single-line if with no brackets or multi-line with brackets.